src/main/resources/application.conf

```diff
@@ -1,5 +1,16 @@
+serverAddress {
```

Comment:
Not a blocker but I would change all names config file to hyphen-case (`server-address`) as it used eg. in Akka's reference.conf: https://github.com/akka/akka/blob/master/akka-actor/src/main/resources/reference.conf